app/models/foreman_docker/service/actions/container/start.rb

```diff
@@ -0,0 +1,30 @@
+module ForemanDocker
+  module Service
+    module Actions
+      module Container
+        class Start < ::Actions::EntryAction
+          def plan(container)
+            action_subject(container)
+            plan_self(:container => container.id)
+          end
+
+          def run
+            container = ::Container.find(input[:container])
+            started = container.compute_resource.create_container(container.parametrize)
+            container.update_attribute(:uuid, started.id) if started
+            started
+          end
+
+          def humanized_name
+            _('Start')
+          end
+
+          def finalize
+            container = ::Container.find(input[:container])
+            Rails.logger.info("[Docker] container #{container.name} successfully started")
```

Comment:
We could probably make better logging in general in the foreman-tasks, so that it would log the info messages about the task (including the information about the action subject)